crates/ironrdp-connector/src/lib.rs

```diff
@@ -72,6 +72,10 @@ impl Credentials {
 pub struct Config {
     /// The initial desktop size to request
     pub desktop_size: DesktopSize,
+    /// The initial desktop scale factor to request.
+    ///
+    /// This becomes the `desktop_scale_factor` in the [`TS_UD_CS_CORE`](gcc::ClientCoreOptionalData) structure.
```

Comment:
Maybe let's remove this option completely if currently only `0` scale factor is supported?

Reply:
All values are supported, the feature is just not hooked up in the `ironrdp-client` currently. I think it's best to leave here, since it's already properly hooked into the correct place in the Client Core Data pdu.